.github/workflows/code-checker.yaml

```diff
@@ -111,6 +111,39 @@ jobs:
           fi
           brew install boost libomp ninja tbb ccache
           make check-proto-product && make verifiers
+  centos:
+    name: Code Checker CentOS 7
+    # Run in centos docker 
+    runs-on: ubuntu-18.04
+    timeout-minutes: 60
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache CCache Volumes
+        uses: actions/cache@v2
+        with:
+          path: .docker/amd64-centos7-ccache
+          key: centos7-ccache-${{ hashFiles('internal/core/**') }}
+          restore-keys: centos7-ccache-
+      - name: Cache Third Party
+        uses: actions/cache@v2
+        with:
+          path: .docker/thirdparty
+          key: centos7-thirdparty-${{ hashFiles('internal/core/**') }}
+          restore-keys: centos7-thirdparty-
+      - name: Cache Go Mod Volumes
+        if: ${{  matrix.os == 'ubuntu-18.04' }}
```

Comment:
Should we change this to centos?

Reply:
Need to remove instead of change to centos, I have already set up centos on separate job already.